### PR TITLE
modifications required for new MIRA and Kinetic

### PR DESCRIPTION
**Pull request by marc-hanheide**
plus setting MIRA_PATH from setup.bash.

Still to be decided if this should be pulled into main branch or if it should be released from this new branch.

### CODE REVIEW

**Comment by marc-hanheide** [comment]
@hawesie @Jailander you may be interested in this. I have released a mira version for Ubuntu Xenial (based on the latest MIRA release) into our new kinetic repos (https://github.com/LCAS/rosdistro/wiki), and have made the modifications to use the the ROS drivers on top of that. Fingers crossed.

File: scitos_drivers/package.xml [hunk 1]
```diff
@@ -6,13 +6,13 @@
...
 
-  <maintainer email="cburbridge@gmail.com">Chris Burbridge</maintainer>
+  <maintainer email="marc@hanheide.net">Marc Hanheide</maintainer>
 
   <license>GPL</license>
 
   <url type="website">http://github.com/strands-project/scitos_drivers</url>
 
-  <author email="cburbridge@gmail.com">Chris Burbridge</author>
+  <author email="marc@hanheide.net">Marc Hanheide</author>
```
Comment (cburbridge) [inline scitos_drivers/package.xml]: I agree that unfortunately I can't maintain this any more, but I think I should still be listed as author, instead just add another author tag?

Comment (marc-hanheide) [reply scitos_drivers/package.xml]: yes, sorry, @cburbridge, that was utter non-sense I made in an attempt to not annoy you with emails. But that clearly went the wrong way.

**Comment by marc-hanheide** [comment]
I think this can be merged, now. Any objections, @hawesie @cburbridge ?

**Comment by hawesie** [comment]
LGTM

**Pull request merged**